### PR TITLE
Bump code coverage

### PR DESCRIPTION
**Pull request by RussKie**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/extensions/pull/5700)

### CODE REVIEW

File: src/Libraries/Microsoft.Extensions.Caching.Hybrid/Microsoft.Extensions.Caching.Hybrid.csproj
Comment (RussKie) [inline src/Libraries/Microsoft.Extensions.Caching.Hybrid/Microsoft.Extensions.Caching.Hybrid.csproj:30]:
```suggestion
    <MinCodeCoverage>86</MinCodeCoverage>
```

**Comment by danmoseley** [comment]
Sorry @RussKie thought this was an old PR but you're working on it right now.

Q: if this "code coverage threshold" system seems to be working well in dotnet/extensions, is it something interesting to start experimenting with in dotnet/runtime, in the libraries? to my knowledge, they only have the ability to gather coverage, not hold thresholds. We all know the reservations and limitations of code coverage and code coverage thresholds, but I'm wondering whether the experiences in this repo might suggest it's valuable for at least some of those libraries. @joperezr @ericstj  @stephentoub

**Comment by eiriktsarpalis** [comment]
> > if this "code coverage threshold" system seems to be working well in dotnet/extensions, is it something interesting to start experimenting with in dotnet/runtime, in the libraries?
> 
> Honestly I'm finding this system to be quite annoying.

Agreed. I find the code coverage reports to be useful, but blocking PRs on an arbitrary threshold feels counterproductive.

**Comment by SteveSandersonMS** [comment]
> Agreed. I find the code coverage reports to be useful, but blocking PRs on an arbitrary threshold feels counterproductive.

Also agreed.

In some rare cases it might be helpful to catch cases where code coverage dropped unexpectedly, but my sense is that most of the time it would go down only for good and valid reasons.

If the bot simply posted the coverage delta as a comment, that would be enough to help us to notice if anything weird is going on, and perhaps would make it easier for reviewers to ask for an explanation or changes if new code seriously lacks coverage.

It's not fun to have to make edits to the coverage thresholds in csproj files when you know it's just bureaucracy and doesn't reflect anything to do with the product's actual quality.

**Comment by RussKie** [comment]
Bump, still waiting for an approval.

**Review by sebastienros — APPROVED** [review]

**Pull request merged**
